### PR TITLE
Russian locale

### PR DESCRIPTION
**Pull request by contributorpw**

### CODE REVIEW

File: i18n/ru.toml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+few = "{{ .Count }} тега"
+many = "{{ .Count }} тегов"
+other = "{{ .Count }} тегов"
+
+[allCategories]
+other = "Все категории"
+
+[category]
+one = "1 категория"
+few = "{{ .Count }} категории"
```
Comment (g1eny0ung) [inline i18n/ru.toml:27]: few and many are not needed.

**Review by g1eny0ung — COMMENTED** [review]
The rest LGTM.

File: i18n/ru.toml (hunk 1 above)
Comment (contributorpw) [reply i18n/ru.toml:27]: Unfortunately I can't build a fine site without `many`. The Hugo logs warnings with "ru" settings.

Comment (g1eny0ung) [reply i18n/ru.toml:27]: I have not encountered this situation 🤔, but it's safe to remain them. Yes, we can clear the `few`.

Comment (contributorpw) [reply i18n/ru.toml:27]: Sorry

```shell
Change detected, rebuilding site.
2021-05-12 13:19:10.422 +0300
i18n changed "/raid/user/github.com/contributorpw/hugo-theme-dream/i18n/ru.toml": WRITE
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
Total in 46 ms
```

**Comment by contributorpw** [comment]
There is my googling research about plural items of the Hugo localization:
* [i18n: plural tags (other than one) are not working · Issue #4889 · gohugoio/hugo](https://github.com/gohugoio/hugo/issues/4889)
* [Plural Rules - CLDR - Unicode Common Locale Data Repository](http://cldr.unicode.org/index/cldr-spec/plural-rules)

**Comment by g1eny0ung** [comment]
> There is my googling research about plural items of the Hugo localization:
> 
> * [i18n: plural tags (other than one) are not working · Issue #4889 · gohugoio/hugo](https://github.com/gohugoio/hugo/issues/4889)
> * [Plural Rules - CLDR - Unicode Common Locale Data Repository](http://cldr.unicode.org/index/cldr-spec/plural-rules)

Now I understand. All LGTM. Thank you for your patience to explain me. 😆

**Pull request merged**